docs/api/spaces-management/copy_saved_objects.asciidoc

```diff
@@ -34,37 +35,59 @@ You can request to overwrite any objects that already exist in the target space
 
 `objects`::
   (Required, object array) The saved objects to copy.
+
+.Properties of `objects`
+[%collapsible%open]
+=====
   `type`:::
     (Required, string) The saved object type.
+
   `id`:::
     (Required, string) The saved object ID.
+=====
 
 `includeReferences`::
   (Optional, boolean) When set to `true`, all saved objects related to the specified saved objects will also be copied into the target spaces. The default value is `false`.
 
 `overwrite`::
   (Optional, boolean) When set to `true`, all conflicts are automatically overidden. When a saved object with a matching `type` and `id` exists in the target space, that version is replaced with the version from the source space. The default value is `false`.
 
-
+[role="child_attributes"]
 [[spaces-api-copy-saved-objects-response-body]]
 ==== {api-response-body-title}
 
 `<space_id>`::
   (object) An object that describes the result of the copy operation for the space. Includes the dynamic keys in the response.
+
+.Properties of `<space_id>`
+[%collapsible%open]
+=====
   `success`:::
     (boolean) The copy operation was successful. When set to `false`, some objects may have been copied. For additional information, refer to the `successCount` and `errors` properties.
+
   `successCount`:::
     (number) The number of objects that successfully copied.
+
   `errors`:::
-    (Optional, array) The errors that occurred during the copy operation. When errors are reported, the `success` flag is set to `false`.v
+    (Optional, array) The errors that occurred during the copy operation. When errors are reported, the `success` flag is set to `false`.
+=====
+
```

Comment:
I think you need a "+" here to embed the collapsible section in the definition list:
```suggestion
+
```